https://github.com/alsa-project/alsa-plugins/commit/ed6684849298

ALSA lib pcm_oss.c:863:(_snd_pcm_oss_open) Unknown field port

--- oss/50-oss.conf.orig	2018-04-03 07:02:58 UTC
+++ oss/50-oss.conf
@@ -5,7 +5,7 @@ pcm.oss {
 		default "/dev/dsp"
 	}
 	type oss
-	port $DEVICE
+	device $DEVICE
 	hint {
 		show {
 			@func refer
